src/components/icon/icon.svelte

```diff
@@ -65,7 +65,7 @@
     flex-shrink: 0;
 
     & .icon,
-    :global svg {
+    :global(svg) {
```

Comment:
do we actually want this?

Reply:
I think yes. Why not?

Reply:
Doesn't it not need to be a child of this component though - that selector is for 
```css
svg {
  /* global svg styles */
```

So if I have an SVG which isn't in this component it'll still get these styles?

Reply:
No, because it's still nested within `.leoIcon`, so the output is:

```css
.leoIcon .icon,
.leoIcon svg {
  width: 100%;
  height: 100%;
}
```

Reply:
:facepalm: